### PR TITLE
add create and get tests

### PR DESCRIPTION
**Pull request by yosef32**

### CODE REVIEW

File: service/permission_test.go [hunk 1]
```diff
@@ -1,21 +1,262 @@
...
+			defer conn.Close()
+
+			t.Parallel()
+
+			client := pb.NewPermissionClient(conn)
+			permissionObject, err := client.CreatePermission(tt.args.ctx, tt.args.req)
+
+			// Unanticipated error - isn't related to tt.wantErr
+			if err != nil {
+				t.Fatalf("DownloadService.Download() error = %v, wantErr %v", err, tt.wantErr)
```
Comment (Shahar-Y) [inline service/permission_test.go]: DownloadService?

File: service/permission_test.go [hunk 2]
```diff
@@ -1,21 +1,262 @@
...
+					Override: false,
+					AppID:    appID,
+				},
+			},
+			wantErr: false,
+		},
+	}
+
+	for _, tt := range tests {
+		tt := tt
```
Comment (Shahar-Y) [inline service/permission_test.go:116]: ?

**Review by Shahar-Y — CHANGES_REQUESTED** [review]

File: service/permission_test.go (hunk 2 above)
Comment (yosef32) [reply service/permission_test.go:116]: this is the format on all tests in go

File: service/permission_test.go (hunk 1 above)
Comment (yosef32) [reply service/permission_test.go]: done

**Pull request merged**